app/Filament/App/Resources/ServerResource/Pages/ListServers.php

```diff
@@ -45,9 +45,19 @@ public function table(Table $table): Table
                     ->relationship('egg', 'name', fn (Builder $query) => $query->whereIn('id', $baseQuery->pluck('egg_id')))
                     ->searchable()
                     ->preload(),
+                SelectFilter::make('owner')
+                    ->relationship('user', 'username', fn (Builder $query) => $query->whereIn('id', $baseQuery->pluck('owner_id')))
+                    ->searchable()
+                    ->visible(fn () => $this->activeTab === 'other')
```

Comment:
Filter should also be available on "All Servers".

```suggestion
                    ->hidden(fn () => $this->activeTab === 'my')
```